source/_data/glossary.yml

```diff
@@ -69,15 +69,16 @@
     devices in the configuration.
 
 - term: Device
-  definition: >
-    A device is a named collection of entities that all represent the same
-    physical/logical unit, which can do or observe something. An example,
-    for a device would be a smart plug named 'Coffee Machine' which provides
-    a `switch` entity plus one or more `sensor` entities for power monitoring
-    or similar.
+  definition: >    
+    A device is a model representing a physical or logical unit that contains entities and has its own control unit.
```

Comment:
What is a "control unit" ?

Reply:
I don't know :-). I found this in the documentation: (_A device in Home Assistant represents either a physical device that has its own control unit_) and added it because I though it was a mandatory component of any device.